### PR TITLE
Send `SecretKey` on start

### PR DESCRIPTION
**Pull request by suhr**

### CODE REVIEW

File: src/tox.rs [hunk 1]
```diff
@@ -529,6 +529,15 @@ fn tox_loop(
 
     let mut tox = Tox::new(tox_options, None).unwrap();
 
+    {
+        let sk = tox.get_secret_key();
+        let ev = Event::SecretKey {
+            secret_key: format!("{}", sk)
+        };
+
+        drop(answer_tx.try_send(Answer::Event(ev)))
```
Comment (kpp) [inline src/tox.rs:538]: why try_send?

**Comment by kpp** [comment]
Also we have to get SK from /ws/SK_IN_HEX uri not from /SK_IN_HEX

File: src/tox.rs (hunk 1 above)
Comment (kpp) [reply src/tox.rs:538]: Also I thought you'd send it in main right after `let (sink, stream) = s.split();`

Comment (suhr) [reply src/tox.rs:538]: Because this is a tokio channel: https://docs.rs/tokio/0.1.20/tokio/sync/mpsc/struct.Sender.html#method.try_send

Comment (suhr) [reply src/tox.rs:538]: `Tox` generates the secret key.

Comment (kpp) [reply src/tox.rs:538]: OK

**Review by kpp — APPROVED** [review]

**Pull request merged**